src/js/player.js

```diff
@@ -2836,19 +2804,38 @@ class Player extends Component {
    *
    * @listens keydown
    */
-  handleKeyPress(event) {
-
-    if (this.options_.userActions && this.options_.userActions.hotkeys && (this.options_.userActions.hotkeys !== false)) {
+  handleKeyDown(event) {
+    const {userActions} = this.options_;
 
-      if (typeof this.options_.userActions.hotkeys === 'function') {
+    // Bail out if hotkeys are not configured.
+    if (!userActions || !userActions.hotkeys) {
+      return;
+    }
 
-        this.options_.userActions.hotkeys.call(this, event);
+    const activeEl = this.el_.ownerDocument.activeElement;
+    const activeTag = activeEl && activeEl.tagName.toLowerCase();
 
-      } else {
+    // Sometimes, people put forms in their player. In general, we do not want
+    // to activate hotkey handling when the focus is on one of these elements.
+    const hotkeysDisallowed = [
+      'button',
+      'datalist',
+      'input',
+      'optgroup',
+      'option',
+      'select',
+      'textarea'
+    ];
 
-        this.handleHotkeys(event);
+    // Bail out if the user is focused on an interactive form element.
+    if (activeTag && hotkeysDisallowed.indexOf(activeTag) !== -1) {
```

Comment:
I think this would prevent hotkeys from working in any of the control bar buttons, no?

Reply:
I don't think so, I did try that, but I'll verify.

Reply:
You're right. I must have added this filtering before actually verifying it! D'oh!

Reply:
I guess you can verify if the element is contained in the control bar and then skip this check

Reply:
🤔

Reply:
Well, buttons are generally only activated on Space/Enter, which we pass through to the browser anyway. Other keys (e.g. m, f, k, etc) are unlikely to affect a button. Seems safe to allow buttons and blacklist them if it becomes an issue for whatever reason.

Reply:
Potentially, though, you can also define buttons as an `input` tag. It may be worth doing the control bar check regardless of which elements we disallow.

Reply:
True, true. The control bar check seems dirty, though. I'll ponder it.